text/src/swam/witx/parser/Lexical.scala

```diff
@@ -0,0 +1,31 @@
+package swam
+package witx
+package parser
+
+import fastparse._
+import NoWhitespace._
+
+/**
+  * @author Javier Cabrera-Arteaga on 2020-03-18
+  */
+object Lexical {
```

Comment:
Can't this be shared between the `wat` and `witx` parsers?

Reply:
I will check this